### PR TITLE
added Sign up page

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

**Review by Jia-A — COMMENTED** [review]
Looks good to me.

File: pages/signup.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <h2 class="heading">Sign up</h2>
+            <form>
+                <div>
+                    <label class="label">Email address</label>
+                    <br>
+                    <input class="text-input" type="email" placeholder="adarshbalika@gmail.com">
+                </div>
+                <div>
+                    <label class="label">Password</label>
+                    <br>
```
Comment (deepak29-git) [inline pages/signup.html:48]: avoid br tag and use margin or padding for that

**Review by deepak29-git — COMMENTED** [review]
Everything looks good

File: pages/signup.html (hunk 1 above)
Comment (shivanigangadharan) [reply pages/signup.html:48]: Noted. 👍

**Pull request merged**